### ISSUE TITLE
MGI data sources  out of date

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-site#2310]
Hi @kltm 

This location: 
https://github.com/geneontology/go-site/blob/master/metadata/datasets/mgi.yaml

still has MGI as sources for GPADs ~and GPIs~. Should we change this as well? The link to the GPAD goes nowhere.

**Comment by kltm** [comment]
@pgaudet I'm not quite following: the information in that file tells us where to get MGI files for the main pipeline and is the correct current information. They would be updated once https://github.com/geneontology/go-site/issues/2285 is closed.

**Comment by kltm** [comment]
Talking to @pgaudet ; we're just talking about the "upstream" GPAD, which does not exist and is not used at our end.

**Comment by pgaudet** [comment]
Discussing with @kltm 

This stanza is wrong, so I will delete it for now

 -
   id: mgi.gpad
   label: "mgi gpad file"
   description: "gpad file for mgi from Mouse Genome Informatics"
   url: https://current.geneontology.org/annotations/mgi.gpad.gz
   type: gpad
   dataset: mgi
   submitter: mgi
   compression: gzip
   source: http://www.informatics.jax.org/downloads/reports/mgi.gpad.gz
   entity_type:
   status: active
   species_code: Mmus
   taxa:
    - NCBITaxon:10090


Thanks, Pascale
